seed/celery.py

```diff
@@ -31,7 +31,7 @@ def on_configure(self):
             pass
 
 
-app = Celery('seed')
+app = Celery('seed', broker='redis://db-redis:6379/')
```

Comment:
without declaring `broker` celery transport defaults to `amqp://guest:**@localhost:5672/`

```
seed_celery    | [2022-01-24 23:24:16,662: WARNING/MainProcess] No hostname was supplied. Reverting to default 'localhost'
seed_celery    |
seed_celery    |  -------------- celery@ea98cb440398 v5.2.2 (dawn-chorus)
seed_celery    | --- ***** -----
seed_celery    | -- ******* ---- Linux-5.10.47-linuxkit-x86_64-with 2022-01-24 23:24:16
seed_celery    | - *** --- * ---
seed_celery    | - ** ---------- [config]
seed_celery    | - ** ---------- .> app:         seed:0x7f5eece31a60
seed_celery    | - ** ---------- .> transport:   amqp://guest:**@localhost:5672//
seed_celery    | - ** ---------- .> results:     redis://db-redis:6379/1
seed_celery    | - *** --- * --- .> concurrency: 2 (prefork)
seed_celery    | -- ******* ---- .> task events: ON
seed_celery    | --- ***** -----
seed_celery    |  -------------- [queues]
seed_celery    |                 .> celery           exchange=celery(direct) key=celery
seed_celery    |
seed_celery    | [2022-01-24 23:24:16,904: ERROR/MainProcess] consumer: Cannot connect to amqp://guest:**@127.0.0.1:5672//: [Errno 111] Connection refused.
```